### ISSUE TITLE
Unable to login to Cync account

### ISSUE THREAD

**Issue by cure1245** [nikshriv/cync_lights#18]
Some type of error in get_cync_config:

```
2022-10-11 17:17:09.830 ERROR (MainThread) [custom_components.cync_lights.config_flow] Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/cync_lights/config_flow.py", line 101, in async_step_two_factor_code
    info["data"]["cync_config"] = await self.cync_hub.get_cync_config()
  File "/config/custom_components/cync_lights/cync_hub.py", line 582, in get_cync_config
    'switch_id':str(device['switchID']),
KeyError: 'switchID'
```

**Comment by halfskye** [comment]
I'm experiencing the same error and am currently unable to log into Cync with the integration.

**Comment by BacchusIX** [comment]
Same error

First occurred: 5:42:33 PM (2 occurrences)
Last logged: 5:43:16 PM

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/cync_lights/config_flow.py", line 101, in async_step_two_factor_code
    info["data"]["cync_config"] = await self.cync_hub.get_cync_config()
  File "/config/custom_components/cync_lights/cync_hub.py", line 582, in get_cync_config
    'switch_id':str(device['switchID']),
KeyError: 'switchID'

**Comment by cure1245** [comment]
If there is a way to attach a debugger to this I wouldn't mind taking a stab at it.

**Comment by nikshriv** [comment]
Working on it. It's been a busy week, haven't had a chance to debug this week. Should be updating in the next day or two to fix this issue among others.

**Comment by cure1245** [comment]
Issue confirmed fixed in commit 46042f89. Thanks!
